webapp/src/components/sidebar/sidebarBoardItem.tsx

```diff
@@ -92,10 +93,14 @@ const SidebarBoardItem = React.memo((props: Props) => {
     const displayTitle: string = board.title || intl.formatMessage({id: 'Sidebar.untitled-board', defaultMessage: '(Untitled Board)'})
     const boardViews = sortBoardViewsAlphabetically(views.filter((view) => view.parentId === board.id))
 
+    console.log('####################################################################')
+    console.log(props.activeViewId)
+    console.log('####################################################################')
+
```

Comment:
I think these should be removed?

Reply:
@hahmadia Harshil just added those for me to add the active indicator on the view as well.